README.md

```diff
@@ -23,6 +23,15 @@ A service worker implementation of the behavior defined in a page&#x27;s App Cac
                 [Demo](projects/sw-appcache-behavior#demo) •
                 [API](projects/sw-appcache-behavior#api)
 
+### sw-offline-google-analytics
+A service worker implementation of the behavior defined in a page&#x27;s App Cache manifest.
```

Comment:
That looks to be an outdated revision—the current version in the branch should have the correct wording.